src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,79 @@
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../types';
+
+interface Props {
+  person: Person
+}
+
+enum PersonSex {
+  Male = 'm',
+  Female = 'f',
+}
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
```

Comment:
It's not a link, it's a person or person row component as it's not only link

Reply:
I named it like that just because there's a task..
![image_2024-01-07_21-15-37](https://github.com/mate-academy/react_people-table-basics/assets/147810951/e72ffee8-c58d-44f2-a028-0685d025b410)